src/social_auth/backends/__init__.py

```diff
@@ -247,9 +247,7 @@ class BaseAuth(object):
 
     def __init__(self, request, redirect):
         self.request = request
-        # Use request because some auth providers use POST urls with needed
-        # GET parameters on it
-        self.data = request.REQUEST
+        self.data = request.GET.update(request.POST)
```

Comment:
Let's not modify this dict here. Could you make a copy first?

Reply:
you right